src/fu-daemon.c

```diff
@@ -1990,33 +1990,43 @@ fu_daemon_daemon_method_call(GDBusConnection *connection,
 		return;
 	}
 	if (g_strcmp0(method_name, "GetBiosAttrs") == 0) {
-		g_autoptr(FuMainAuthHelper) helper = NULL;
+		gboolean authenticate = fu_engine_request_get_feature_flags(request) &
+					FWUPD_FEATURE_FLAG_ALLOW_AUTHENTICATION;
+
+		g_debug("Called %s", method_name);
+		if (!authenticate) {
+			g_autoptr(FuBiosAttrs) attrs =
+			    fu_context_get_bios_attrs(fu_engine_get_context(self->engine));
+			val = fu_bios_attrs_to_variant(attrs,
+						       fu_engine_request_get_device_flags(request) &
+							   FWUPD_DEVICE_FLAG_TRUSTED);
```

Comment:
I'm a bit confused with this; is this ever going to be true?

Reply:
I think you're right.  This is a vestige from an earlier version of this patch that doesn't make sense anymore.